src/main/java/org/codehaus/groovy/tools/FileSystemCompiler.java

```diff
@@ -77,18 +73,6 @@ public void compile(File[] files) throws Exception {
         unit.compile();
     }
 
-    public static void displayHelp(final Options options) {
```

Comment:
Okay. Note that the Options class is no longer used anywhere in `FileSystemCompiler` so this would be purely for binary compatibility.

How do you want to do this? I'm working on master, so the method is removed in the PR. Do you want a separate PR for 2_5_X with the method reinstated or will you manually reinstate on the 2_5_X branch when cherrypicking?